Px.Utils/Validation/ContentValidation/ContentValidator.UtilityMethods.cs

```diff
@@ -238,34 +211,66 @@ private static List<ValidationFeedbackItem> ProcessContentDimensionValue(
         /// <param name="validator">Object that provides information of the ongoing content validation process</param>
         /// <param name="dimensionName">Name of the dimension currently being processed</param>
-        /// <returns></returns>
-        private static List<ValidationFeedbackItem> ProcessDimension(
+        /// <returns>Returns a <see cref="ValidationFeedback"/> object containing information of missing dimension entries</returns>
+        private static ValidationFeedback ProcessDimension(
             string[] keywords,
             string dimensionTypeKeyword,
             ValidationStructuredEntry[] entries, 
             string language,
             ContentValidator validator,
             string dimensionName)
         {
-            List<ValidationFeedbackItem> feedbackItems = [];
+            ValidationFeedback feedbackItems = [];
             foreach (string keyword in keywords)
             {
-                ValidationFeedbackItem? keywordFeedback = FindDimensionRecommendedKey(entries, keyword, language, validator, dimensionName);
+                KeyValuePair<ValidationFeedbackKey, ValidationFeedbackValue>? keywordFeedback = FindDimensionRecommendedKey(entries, keyword, language, validator, dimensionName);
                 if (keywordFeedback is not null)
                 {
-                    feedbackItems.Add((ValidationFeedbackItem)keywordFeedback);
+                    feedbackItems.Add((KeyValuePair<ValidationFeedbackKey, ValidationFeedbackValue>)keywordFeedback);
                 }
             }
 
             if (language == validator._defaultLanguage)
             {
-                ValidationFeedbackItem? variableTypeFeedback = FindDimensionRecommendedKey(entries, dimensionTypeKeyword, language, validator, dimensionName);
+                KeyValuePair<ValidationFeedbackKey, ValidationFeedbackValue>? variableTypeFeedback = FindDimensionRecommendedKey(entries, dimensionTypeKeyword, language, validator, dimensionName);
                 if (variableTypeFeedback is not null)
                 {
-                    feedbackItems.Add((ValidationFeedbackItem)variableTypeFeedback);
+                    feedbackItems.Add((KeyValuePair<ValidationFeedbackKey, ValidationFeedbackValue>)variableTypeFeedback);
                 }
             }
             return feedbackItems;
         }
+
+        /// <summary>
+        /// Cleans a list of strings from new line characters and quotation marks.
+        /// </summary>
+        /// <param name="input">The input array of strings to clean</param>
+        /// <param name="syntaxConf>The syntax configuration for the PX file. The syntax configuration is represented by a <see cref="PxFileSyntaxConf"/> object.</param>
+        /// <returns>A list of strings that are the input strings cleaned from new line characters and quotation marks</returns>
+        internal static List<string> CleanListOfStrings(List<string> input, PxFileSyntaxConf syntaxConf)
```

Comment:
input.Select(s => SyntaxValidationUtilityMethods.CleanString(s, syntaxConf)).ToList(); Could be a one line repelacement for this function.